src/app/list.component.html

```diff
@@ -1,8 +1,10 @@
+<stats [init]="todos"></stats>
```

Comment:
it will give you undefined value first time if you console value of init in stats.component.ts. Fix this.

Reply:
ok sir!